xontrib/fzf-widgets.xsh

```diff
@@ -6,11 +6,12 @@ from xonsh.completers.path import complete_path
 from xonsh.platform import ptk_shell_type
 from prompt_toolkit.keys import Keys
 
-__all__ = ()
+__all__ = ['Keys']
 
 def get_fzf_binary_name():
-    if 'TMUX' in ${...}:
-        return 'fzf-tmux'
+    fzf_tmux_cmd = 'fzf-tmux'
+    if 'TMUX' in ${...} and $(which fzf_tmux_cmd):
```

Comment:
$(..) form does _not_ expect python vars. This line translates to :
`which "fzf_tmux_cmd"`... not what we have in mind.
PR incoming.